toonz/sources/toonz/onionskinmaskgui.cpp

```diff
@@ -102,6 +125,18 @@ void OnioniSkinMaskGUI::addOnionSkinCommand(QMenu *menu, bool isFilmStrip) {
         menu->connect(extendOnionSkinToScene, SIGNAL(triggered()), &switcher,
                       SLOT(setWholeScene()));
       }
+      QAction *clearFixedOnionSkins = menu->addAction(
+          QString(QObject::tr("Clear All Fixed Onion Skin Markers")));
+      menu->connect(clearFixedOnionSkins, SIGNAL(triggered()), &switcher,
+                    SLOT(clearFOS()));
+      QAction *clearRelativeOnionSkins = menu->addAction(
+          QString(QObject::tr("Clear All Relative Onion Skin Markers")));
+      menu->connect(clearRelativeOnionSkins, SIGNAL(triggered()), &switcher,
+                    SLOT(clearMOS()));
+      QAction *clearAllOnionSkins =
+          menu->addAction(QString(QObject::tr("Clear All Onion Skin Markers")));
+      menu->connect(clearAllOnionSkins, SIGNAL(triggered()), &switcher,
+                    SLOT(clearOS()));
```

Comment:
For simplification, how about showing only `Clear All Onion Skin Markers` if there is only one type of the marker in the current scene?